composer/core/time.py

```diff
@@ -200,6 +200,22 @@ def unit(self) -> TimeUnit:
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}({self.value}, {self.unit})"
 
+    def __str__(self) -> str:
+        return f"{self.value}{self.unit.value}"
+
+    def to_timestring(self):
+        """Get the time-string representation.
+
+        For example:
+
+        >>> Time(5, TimeUnit.EPOCH).to_timestring()
```

Comment:
Does this need `.. doctest::` syntax?

Reply:
If there is no `..testsetup` needed, then no ``..doctest`` is needed -- you can do the `>>>` directly.